docs/dev/table/connect.md

```diff
@@ -1442,6 +1442,8 @@ CREATE TABLE MyUserTable (
   'format.field-delimiter' = ';',         -- optional: field delimiter character (',' by default)
   'format.line-delimiter' = '\r\n',       -- optional: line delimiter ("\n" by default; otherwise
                                           -- "\r" or "\r\n" are allowed)
+  'format.disable-quote-character' = true,-- optional: disabled quote character for enclosing field values (false by default)
```

Comment:
Could you update remaining languages? Java/Scala/Python?

Moreover in SQL DDL the value of a property must be quoted.

Reply:
Thanks, got it. My changes do not support python.